src/main/java/org/broadinstitute/consent/http/service/dao/DarCollectionServiceDAO.java

```diff
@@ -42,7 +40,7 @@ public DarCollectionServiceDAO(DatasetDAO datasetDAO, ElectionDAO electionDAO, J
    * Find all Dar Collection + Dataset combinations that are available to the user.
    *    - Admins have all available to them
    *    - Chairs can only create elections for datasets in their DACs
-   *
+   * <p>
```

Comment:
accidental?

Reply:
That was intentional, it was an Intellij suggestion.